bubblesub/cmd/file_properties.py

```diff
@@ -293,17 +293,16 @@ def _load(self) -> None:
 
         self._api.subs.meta.clear()
 
+        wrap_mode_combo_box = self._options_group_box.wrap_mode_combo_box
+        ycbcr_matrix_combo_box = self._options_group_box.ycbcr_matrix_combo_box
```

Comment:
Rather than extracting widgets to variables, I'd like to extract their data instead. And rather than just two of them that happen to have wordy names, treat all of them the same and extract everything to variables.

Reply:
Ok, so I will extract every data to variables